user_docs/en/userGuide.t2t

```diff
@@ -485,8 +485,9 @@ The NVDA menu allows you to control NVDA's settings, access help, save/revert yo
 To get to the NVDA menu from anywhere in Windows while NVDA is running, you may do any of the following:
 - press ``NVDA+n`` on the keyboard.
 - Perform a 2-finger double-tap on the touch screen.
-- Access the system tray by pressing the Windows logo key+B, DownArrow to the NVDA icon, and press ``enter``.
-- Alternatively, access the system tray by pressing ``Windows+b``, DownArrow to the NVDA icon, and open the Context menu by pressing the ``applications`` key located next to the right control key on most keyboards. On a keyboard without an ``applications`` key, press ``shift+F10`` instead.
+- Access the system tray by pressing ``windows+b``, ``downArrow`` to the NVDA icon, and press ``enter``.
+- Alternatively, access the system tray by pressing ``windows+b``, ``downArrow`` to the NVDA icon, and open the Context menu by pressing the ``applications`` key located next to the right control key on most keyboards.
+On a keyboard without an ``applications`` key, press ``shift+F10`` instead.

```

Comment:
Hi, just fixing up some capitalization to be more consistent with the rest of the user guide.

```suggestion
- Access the system tray by pressing ``Windows+b``, ``downArrow`` to the NVDA icon, and press ``enter``.
- Alternatively, access the system tray by pressing ``Windows+b``, ``downArrow`` to the NVDA icon, and open the context menu by pressing the ``applications`` key located next to the right control key on most keyboards.
On a keyboard without an ``applications`` key, press ``shift+F10`` instead.
```